src/pal/src/exception/seh-unwind.cpp

```diff
@@ -135,8 +139,11 @@ BOOL VirtualUnwind(CONTEXT *context, KNONVOLATILE_CONTEXT_POINTERS *contextPoint
     unw_context_t unwContext;
     unw_cursor_t cursor;
 
-    // Set the unwind context to the specified windows context
-    WinContextToUnwindContext(context, &unwContext);
+    st = unw_getcontext(&unwContext);
```

Comment:
Do we really need to use the unw_getcontext when we overwrite all the registers few lines later?
I am also worried that calling the unw_init_local before the WinContextToUnwindContext can cause issues with the libunwind on Linux. I am not allowed to look at the sources of this libunwind yet, but I guess that the unw_init_local would initialize the unwinder internal state based on the context passed in and that setting the register values later might not update the state correctly.